test/coverage.spec.ts

```diff
@@ -0,0 +1,275 @@
+import { createFixture } from './fixture';
+import { betterer } from '@betterer/betterer';
+import { BettererCoverageIssues } from '@betterer/coverage';
+import { Fixture } from '@betterer/fixture';
+
+interface Coverage {
+  lines: { total: number; covered: number; skipped: number; pct: number };
+  statements: { total: number; covered: number; skipped: number; pct: number };
+  functions: { total: number; covered: number; skipped: number; pct: number };
+  branches: { total: number; covered: number; skipped: number; pct: number };
+  branchesTrue?: { total: number; covered: number; skipped: number; pct: number };
+}
+
+describe('betterer', () => {
+  describe('total coverage test', () => {
+    let totalResult: Coverage;
+
+    beforeEach(() => {
+      totalResult = {
+        lines: { total: 4931, covered: 227, skipped: 0, pct: 4.6 },
+        statements: { total: 5185, covered: 236, skipped: 0, pct: 4.55 },
+        functions: { total: 1340, covered: 31, skipped: 0, pct: 2.31 },
+        branches: { total: 3025, covered: 83, skipped: 0, pct: 2.74 },
+        branchesTrue: { total: 0, covered: 0, skipped: 0, pct: 100 }
+      };
+    });
+
+    it('should report the total coverage', async () => {
```

Comment:
Would be good to have each test in its own file so that they can be run with as much parallelism as possible. I find this also means you have to write tests in such a way that they are totally independent of each other.

Reply:
At least for me, jest already runs stuff in parallel that's in a different describe. Are separate files really neccessary?